tests/conftest.py

```diff
@@ -411,19 +428,26 @@ def docker_image(boto_model_file, sagemaker_endpoint):
     except docker.errors.NotFound:
         pass
 
+    os.makedirs(os.path.join(docker_dir, "boto"), exist_ok=True)
+
     if boto_model_file is None:
         print("boto_model_file is None, using default model.")
     else:
-        shutil.copy(boto_model_file, "tests/integ-jobs/docker/boto/sagemaker-experiments-2017-07-24.normal.json")
+        shutil.copy(boto_model_file, os.path.join(docker_dir, "boto/sagemaker-experiments-2017-07-24.normal.json"))
 
     subprocess.check_call([sys.executable, "setup.py", "sdist"])
     [sdist_path] = glob.glob("dist/sagemaker-experiments*")
-    shutil.copy(sdist_path, "tests/integ-jobs/docker/smexperiments-0.1.0.tar.gz")
 
-    os.makedirs("tests/integ-jobs/docker/boto", exist_ok=True)
+    shutil.copy(sdist_path, os.path.join(docker_dir, "smexperiments-0.1.0.tar.gz"))
 
+    # may need to configure cred helper in ~/.docker/config.json
+    # {
+    # 	"credHelpers": {
+    # 		"aws_account_id.dkr.ecr.region.amazonaws.com": "ecr-login"
+    # 	}
+    # }
```

Comment:
do we need these?

Reply:
ya this is a prerequisite setup step to get this test to work